src/keyfile.c

```diff
@@ -255,6 +255,8 @@ static void init_pref_groups(void)
 		"extract_filetype_regex", GEANY_DEFAULT_FILETYPE_REGEX);
 	stash_group_add_boolean(group, &search_prefs.replace_and_find_by_default,
 		"replace_and_find_by_default", TRUE);
+	stash_group_add_boolean(group, &editor_prefs.ime_interaction,
+		"editor_ime_inline", TRUE);
```

Comment:
Same here, should be
```C
	stash_group_add_integer(group, &editor_prefs.ime_interaction,
		"editor_ime_interaction", SC_IME_WINDOWED);
```
And better keep the current behavior as default unless we have a compelling set of data that the current default is a problem for everyone.